services/tests/test_services_graphql_api.py

```diff
@@ -246,3 +250,144 @@ def test_normal_user_can_query_own_services_gdpr_api_scopes(
     executed = user_gql_client.execute(query)
 
     assert dict(executed["data"]) == expected_data
+
+
+def _set_service_title_and_description(service):
+    service.set_current_language("fi")
+    service.title = "Service title in finnish"
+    service.description = "Service description in finnish"
+    service.save()
+
+    service.set_current_language("en")
+    service.title = "Service title in english"
+    service.description = "Service description in english"
+    service.save()
+
+
+@pytest.mark.parametrize("use_variable", [False, True])
+@pytest.mark.parametrize("language", ["EN", "FI"])
+def test_service_title_translation(
+    user_gql_client, service_factory, use_variable, language
+):
+    profile = ProfileFactory(user=user_gql_client.user)
+    service = service_factory(service_type=ServiceType.BERTH)
```

Comment:
Service type is deprecated. They shouldn't be used in tests unless the point of the test is to check that they still exist. Here I suppose that's not the case.

Instead of using `service_factory` here, you can just request the `service` fixture.